styles.css

```diff
@@ -0,0 +1,21 @@
+#image{
+  text-align: center;
+  max-width: 100%;
+  height: auto;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h1{
+  text-align: center;
+}
+
+p{
+  text-align: justify;
+  font-size: 20px;
+}
+
+#img-caption{
+  text-align: center;
+}
```

Comment:
please use stylelint to format your CSS files. Reasons to use linters:
1. Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
2. The entire codebase looks like written by one person.
3. Programmers can focus on solving problems, instead of cleaning up the code.
Please use this [link](https://github.com/microverseinc/linters-config) for a how-to setup linters in your local environment guide.